docs/usage/usage.md

```diff
@@ -599,7 +622,7 @@ spec:
   cloudProfile:
     name: aws
   region: eu-central-1
-  secretBindingName: core-aws
+  credentialsBindingName: core-aws
```

Comment:
There is CredentialsBinding named `core-aws` in the examples, can you add one so that the examples are complete.